WcaOnRails/app/views/static_pages/faq.html.erb

```diff
@@ -13,103 +29,14 @@
-        In general, there are not many requirements for a person to participate
-        at a WCA competition. However, all participants of a competition are expected
-        to have a good understanding of the
-        <a href="https://www.worldcubeassociation.org/regulations/">WCA regulations</a>.
-        In addition, we would recommend having a look at
-        <%= link_to "Cubing USA's competitor tutorial", "http://www.cubingusa.com/ctutorial.php", target: "_blank" %>, which gives several nice hints for newcomers.
-      <% end %>
 
-      <%= add_answer "How can I have a WCA Competition in my hometown?" do %>
-        <p>
-          If you are interested in organizing a competition, it's highly recommended to attend at least one or two competitions as a competitor to learn from the experience.
-        </p>
-        <p>
-          WCA Competitions must follow the <%= link_to "WCA Regulations", "https://www.worldcubeassociation.org/regulations/", target: "_blank" %>. The organization team doesn't have to know the regulations by heart, however, a deep knowledge of the basics of the regulations would definitely help in correctly organizing the competition.
-        </p>
-        <p>
-          After that, the organization team must contact a nearby <%= link_to "WCA Delegate", "https://www.worldcubeassociation.org/delegates", target: "_blank" %>. The supervision of a WCA Delegate is required in order to make a competition officially sanctioned by the WCA. The WCA Delegate will help guide the organization on how to host a successful competition.
-        </p>
-        <p>
-          Visit <%= link_to "this guide", "https://www.cubingusa.com/cguide.php", target: "_blank" %> for further information. (Any conflict between the information presented in this supplementary resource and the WCA Regulations is superseded by the WCA Regulations.)
-        </p>
-      <% end %>
-
-      <%= add_answer "What are the WCA accounts for? What is the difference between WCA accounts and WCA profiles?" do %>
-        <p>
-          You are assigned a WCA ID when you first participate in a WCA
-          sanctioned competition. When the results from that competition are uploaded,
-          your WCA profile will be created, with results from every competition you have
-          been to.
-        </p>
-
-        <p>
-          WCA accounts allow you to prove to us who you are. This allows you
-          to add a picture to your WCA profile, and allows you
-          to register for competitions. This will protect you from
-          other people pretending to be you.
-        </p>
-
-        <p>
-          <% if current_user %>
-            You are already logged in with a WCA account.
-          <% else %>
-            You can create an account <%= link_to "here", new_user_registration_path %>.
+      <% t('faq.answers').values.each do |faq| %>
+        <%= add_answer faq[:title] do %>
```

Comment:
Argh that's actually so bad!
I can live with a page being displayed differently depending on the locale, but here it would just mess up the link if the user hasn't the same locale!

Also reading this now that we've had a new item in the FAQ, it makes me think it's an insanely bad idea to iterate through the *translated* keys!
I'd rather display an English FAQ in a foreign text rather than missing a entry because it's not been translated...

I'll change this now in two ways:
- iterate through the English FAQ entries
- generate the anchor from the English title